app/main/routes_devices.py

```diff
@@ -135,10 +135,12 @@ def handle_devices():
     # merge PicoBrewC_Alt and PicoBrewC for /devices experience
     merged_config = server_config()
     if 'PicoBrewC_Alt' in merged_config['aliases']:
+        pico_c_alt = None
```

Comment:
I'm not sure I follow the logic here - the check on 137 makes sure its in merged_config - do you have garbage in your config files?

Reply:
@tmack8001 ?

Reply:
Looks like there is an exception/edge case when `PicoBrewC_Alt` isn't defined in config.yml when a `PicoBrewC` device is deleted from the configuration.

Though this change is only for the GET, which throws this exception if there is no key within `PicoBrewC_Alt`, so yeah this is a partial change that needs to be made. Will add the other logic here as well.

Reply:
fork/clones don't always have maintainer permissions to push -> so I needed to create a new PR against the fork for some additional changes https://github.com/CyberMind-FR/picobrew_pico/pull/1